File: C language/Assignment-6/HCF of two numbers.c
#include<stdio.h>
#include<conio.h>
int main()
{
    int n1,n2,i=1,hcf;
    printf("Enter the first number:");
    scanf("%d",&n1);
    printf("Enter the second number:");
    scanf("%d",&n2);

   while(i<=(n1>n2?n1:n2))
    {
       if(n1%i==0 && n2%i==0 )
       {
          hcf=i;
       }
        i++;
    }
    printf("The HCF of %d and %d is %d",n1,n2,hcf);
    return 0;
}

File: Python MySirG/Assignment-12/hcf of two numbers.py
n1=int(input("Enter the first number:"))
n2=int(input("Enter the second number:"))
if n1<n2:
    hcf=n1
else:
    hcf=n2
i=hcf
while i>=1:
    HCF=i
    i-=1
    if(n1%hcf==0 and n2%hcf==0):
        break;
print("HCF=%d"%HCF)
